desc/vmec.py

```diff
@@ -328,14 +329,23 @@ def save(cls, eq, path, surfs=128, verbose=1):
         ai = file.createVariable("ai", np.float64, ("preset",))
         ai.long_name = "rotational transform coefficients"
         ai[:] = np.zeros((file.dimensions["preset"].size,))
-        # only using up to 10th order to avoid poor conditioning
-        ai[:11] = PowerSeriesProfile.from_values(
-            s_full, eq.iota(r_full), order=10, sym=False
-        ).params
+        if eq.iota is not None:
+            # only using up to 10th order to avoid poor conditioning
+            ai[:11] = PowerSeriesProfile.from_values(
```

Comment:
if `iota` here is already a `PowerSeriesProfile` in `rho`, couldn't we just take its coefficients and divide its mode numbers by 2 to get the corresponding series in `s` exactly? instead of fitting a power series

Reply:
The only downside to your recommendation is it adds more specific logic to maintain. If the underlying profile is already a power series, then the fitting should already work pretty well.